Source/ACE.Server/WorldObjects/Creature_Death.cs

```diff
@@ -266,6 +269,18 @@ private void GenerateTreasure(Corpse corpse)
                     }
                 }
             }
+
+            //If creature didnt have a CreateList, then this creates the loot.
+            if(!hasCreateList && Level > 3)
```

Comment:
Should we not be doing this check regardless of createlist being populated?

Based on what I read of the earlier discussion it would suggest both were evaluated

Reply:
Both were being evaluated when I removed !hasCreateList